app/managers.py

```diff
@@ -0,0 +1,54 @@
+import os
+import sqlite3
+
+
+class ActorManager :
```

Comment:
The class name 'ActorManager' should be in singular form according to the checklist. Consider renaming it to 'Actor'.